secure_tunneling/source/SecureTunnel.cpp

```diff
@@ -102,37 +106,37 @@ namespace Aws
 
         void SecureTunnel::s_OnConnectionComplete(void *user_data)
         {
-            auto *secureTunnel = static_cast<SecureTunnel *>(user_data);
+            auto *secureTunnel = reinterpret_cast<SecureTunnel *>(user_data);
```

Comment:
Why are these `reinterpret_cast` now? You can `static_cast` just fine from `void*` -> known type

Reply:
It was suggested by @DavidOgunsAWS a while ago but I never submitted a PR for it. I honestly haven't read about when I should use which.

Reply:
Undo the change and use `static_cast`.